pkg/sql/exec_util.go

```diff
@@ -2034,12 +2034,6 @@ type queryMeta struct {
 	database string
 }
 
-// cancel cancels the query associated with this queryMeta, by closing the
-// associated stmt context.
-func (q *queryMeta) cancel() {
```

Comment:
This method just calls a similarly named function field and seems like an unnecessary layer.